frontend/website/pages/docs/my-first-transaction.mdx

```diff
@@ -148,7 +148,9 @@ Finally we get to the good stuff, sending our first transaction! Before you send
 
     coda accounts unlock -public-key $CODA_PUBLIC_KEY
 
-For testing purposes, there's already an echo service set up that will immediately refund your payment minus the transaction fees.
+For testing purposes, there's already an echo service set up that will immediately refund your payment minus the transaction fees. You can use the following public keys for the echo bot.
+- 4vsRCVNep7JaFhtySu6vZCjnArvoAhkRscTy5TQsGTsKM4tJcYVc3uNUMRxQZAwVzSvkHDGWBmvhFpmCeiPASGnByXqvKzmHt4aR5uAWAQf3kqhwDJ2ZY3Hw4Dzo6awnJkxY338GEp12LE4x
```

Comment:
```suggestion
- 4vsRCVNep7JaFhtySu6vZCjnArvoAhkRscTy5TQsGTsKM4tJcYVc3uNUMRxQZAwVzSvkHDGWBmvhFpmCeiPASGnByXqvKzmHt4aR5uAWAQf3kqhwDJ2ZY3Hw4Dzo6awnJkxY338GEp12LE4x
```

Reply:
Please only include the "working" one, we're only including the "dead" echo bot because people may have sent transactions there by accident.

Reply:
fixed!